amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseCoordinator.java

```diff
@@ -211,10 +211,9 @@ public void run() {
     @Override
     public void initialize() throws ProvisionedThroughputException, DependencyException, IllegalStateException {
         final boolean newTableCreated =
-                leaseRefresher.createLeaseTableIfNotExists(initialLeaseTableReadCapacity, initialLeaseTableWriteCapacity);
```

Comment:
There should be a unit test in `DynamoDBLeaseCoordinatorTest` to make sure that `createLeaseIfNotExists()` gets called in `initialize()` as part of validating this change.